lib/obstacles/include/obstacles/List.hpp

```diff
@@ -23,13 +23,21 @@
 #include "obstacles/Obstacle.hpp"
 #include "tracefd/File.hpp"
 
+#include "PB_Obstacle.hpp"
+
+
+#define OBSTACLES_MAX_NUMBER 32
```

Comment:
This macro is useful but this parameter should be changeable from cmdline.
Thus I would use something like:
```C
#if !defined(OBSTACLES_MAX_NUMBER)
#define OBSTACLES_MAX_NUMBER 32
#endif
```

Reply:
Done